xml/admin_ceph_gateway.xml

```diff
@@ -3152,6 +3165,43 @@ data sync source: 341c2d81-4574-4d08-ab0f-5a2a7b168028 (us-east)
                   full sync: 0/128 shards
                   incremental sync: 128/128 shards
                   data is caught up with source</screen>
+    <para>
+     The output can differ depending on the sync status. The shards are
+     described as two different types during sync:
+    </para>
+    <variablelist>
+     <varlistentry>
+      <term>Behind shards</term>
+      <listitem>
+       <para>
+        Behind shards are shards that need a full data sync and shards needing
```

Comment:
```suggestion
        Behind shards need a full data sync and shards needing

Reply:
I can't make a bigger suggestion, as @asettle's is blocking the way.

I think her suggestion is an improvement but there is room for more.

I suggest:

«
Shards described as [emphasis]behind[/emphasis] need a full or incremental data-sync, because they are not up to date.
»